src/mirco_setparameters.cpp

```diff
@@ -51,9 +51,23 @@ void MIRCO::SetParameters(double& E1, double& E2, double& LateralLength, double&
   // Composite Poisson's ratio
   CompositePoissonsRatio = CompositeYoungs / (2 * CompositeShear) - 1;
 
-  // Correction factor vector
-  std::vector<double> alpha_con{0.778958541513360, 0.805513388666376, 0.826126871395416,
-      0.841369158110513, 0.851733020725652, 0.858342234203154, 0.862368243479785,
+  // Shape factors (See section 3.3 of https://doi.org/10.1007/s00466-019-01791-3)
+  // These are the shape factors to calculate the elastic compliance correction of the micro-scale
+  // contact constitutive law for various resolutions.
+  // NOTE: Currently MIRCO works for resouluion of 1 to 8. The following vectors store the shape
+  // factors for resolution of 1 to 8.
```

Comment:
Alternatively you could use a map here, something like
```c++
const std::map<int, double> shape_factors_presure{{1,0.961389237917602},{2,0.924715342432435},...};
```

Then you can access the data with `shape_factors_pressure[Resolution]` instead of `shape_factors_pressure[Resolution - 1]`, i.e., you map a resolution value to a pressure.

Reply:
I like the `map` proposal. It makes the assignment of resolution to the value  very explicit.